web/libs/js/main.dash2.js

```diff
@@ -2046,7 +2046,7 @@ $.timelapse.e.on('click','[timelapse]',function(){
                     video=e.video
```

Comment:
i just found out the red lines are because i am using EJS variables inside the js file. `<%- something %>` and that seems to be what causes those red lines.